source/users/delete-or-revoke.txt

```diff
@@ -91,6 +103,66 @@ Manually Delete a User
 
 .. include:: /includes/note-delete-user.rst
 
+Delete a User with a Custom Function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can write a custom function to delete a user. You might want to do this if
+your SDK does not yet support the delete users API. 
+
+:ref:`Create a function <define-a-function>` similar to our example below 
+that uses :guilabel:`Application Authentication`. You might want to incorporate
+error handling in the event that the function does not successfully 
+authenticate, or it cannot delete the calling user. 
+
+For this example function, we have :ref:`created values and secrets for the 
+adminApiPublicKey and adminApiPrivateKey <values-and-secrets>`. 
+We would then add the :ref:`Project and Application IDs 
+<find-your-project-or-app-id>` to the ``apiUrl``.
+
+We can then :ref:`call this function from the SDK 
+<call-function-from-client-sdks>`, passing in an argument. The example
+function below does not take any arguments, and deletes the user who
+calls the function. However, the SDK expects to provide an argument when you 
+call a function, so you can pass in an arbitrary argument like 
+"Delete this user."
```

Comment:
I wonder if we should report this to the SDK folks as a bug -- I feel like you should be able to call functions with no argument. Ought to be an easy fix, I would think?